### ISSUE TITLE
Upgrade fails with the material theme

### ISSUE THREAD

**Issue by nicofrand** [YunoHost-Apps/shaarli_ynh#64]
### Describe the bug

Upgrade to 0.12.x fails and restoration too.

### Context

I am using a theme for Shaarli (material theme).

### Steps to reproduce

* Have a theme downloaded (material for me) in `/var/www/shaarli/tpl/`
* Have the `theme` set in `/var/www/shaarli/data/config.json.php` to `material`
* Upgrade the app to 0.12.x

### Expected behavior

The upgrade works as intended.

### Logs

The logs below were a nice hint:
```bash
2020-12-01 11:38:47,607: WARNING - chmod: cannot access '/var/www/shaarli/tpl/material/dist/fonts/roboto/Apache': No such file or directory
2020-12-01 11:38:47,608: WARNING - chmod: cannot access 'License.txt': No such file or directory
```

https://paste.yunohost.org/raw/gisicapeki

**Comment by lapineige** [comment]
If you remove (make a backup ;) the theme, does it work ?

**Comment by nicofrand** [comment]
I knew I was forgetting something…

**Yes**, it does.

**Comment by lapineige** [comment]
And I guess that '/var/www/shaarli/tpl/material/dist/fonts/roboto/Apache' file exists ?

**Comment by nicofrand** [comment]
Nope! That's probably due to a whitespace which is not escaped:

```bash
root@nicofrand:/home/admin# ls -l /var/www/shaarli/tpl/material/dist/fonts/roboto/
total 2552
-rw-r--r-- 1 shaarli shaarli  11590 Nov 28 15:11 'Apache License.txt'
-rw-r--r-- 1 shaarli shaarli 149644 Nov 28 15:11  Roboto-BlackItalic.ttf
-rw-r--r-- 1 shaarli shaarli 142472 Nov 28 15:11  Roboto-Black.ttf
-rw-r--r-- 1 shaarli shaarli 144700 Nov 28 15:11  Roboto-BoldItalic.ttf
-rw-r--r-- 1 shaarli shaarli 135820 Nov 28 15:11  Roboto-Bold.ttf
-rw-r--r-- 1 shaarli shaarli 145256 Nov 28 15:11  RobotoCondensed-BoldItalic.ttf
-rw-r--r-- 1 shaarli shaarli 141796 Nov 28 15:11  RobotoCondensed-Bold.ttf
-rw-r--r-- 1 shaarli shaarli 144404 Nov 28 15:11  RobotoCondensed-Italic.ttf
-rw-r--r-- 1 shaarli shaarli 145104 Nov 28 15:11  RobotoCondensed-LightItalic.ttf
-rw-r--r-- 1 shaarli shaarli 141384 Nov 28 15:11  RobotoCondensed-Light.ttf
-rw-r--r-- 1 shaarli shaarli 140396 Nov 28 15:11  RobotoCondensed-Regular.ttf
-rw-r--r-- 1 shaarli shaarli 148540 Nov 28 15:11  Roboto-Italic.ttf
-rw-r--r-- 1 shaarli shaarli 145932 Nov 28 15:11  Roboto-LightItalic.ttf
-rw-r--r-- 1 shaarli shaarli 140276 Nov 28 15:11  Roboto-Light.ttf
-rw-r--r-- 1 shaarli shaarli 147876 Nov 28 15:11  Roboto-MediumItalic.ttf
-rw-r--r-- 1 shaarli shaarli 137308 Nov 28 15:11  Roboto-Medium.ttf
-rw-r--r-- 1 shaarli shaarli 145348 Nov 28 15:11  Roboto-Regular.ttf
-rw-r--r-- 1 shaarli shaarli 132376 Nov 28 15:11  Roboto-ThinItalic.ttf
-rw-r--r-- 1 shaarli shaarli 130044 Nov 28 15:11  Roboto-Thin.ttf
```

**Comment by lapineige** [comment]
Thanks for that information !

**Comment by lapineige** [comment]
Should be solved by #63. Feel free to reopen if it's not the case.
